### DISCUSSION TITLE
WPF Devtools in dock

### DISCUSSION THREAD

**Discussion by erimeurer** [cefsharp/CefSharp#4010] [General]
It's possible with CefSharp WPF to render devtools in a dock, I managed to do it in a way that I render inside a handle, but devtools doesn't follow the parent window and doesn't look the way I want.

**Comment by amaitland** [comment]
> I managed to do it in a way that I render inside a handle

What does your code look like currently? There are a number of fairly low level calls needed to resize/move/hide the browser correctly.

**Comment by erimeurer** [comment]
I'm using WindowsFormsHost in WPF
![image](https://user-images.githubusercontent.com/84336182/154756435-937a5400-5edc-4149-83cf-0b701f06a3db.png)
Background where it is blank it resizes along with the window, but the devtools that is inside it does not

**Comment by erimeurer** [comment]
I found where is the code that makes the fitting!
They are events that you have to apply in the browser

**Reply by jean-knapp** [reply]
Would you mind sharing your solution?
